### PR TITLE
Search dev tools usability improvements

### PR DESCRIPTION
**Pull request by rinatkhaziev**
## Description

This PR contains a few UI updates:

1. A copy button is implemented (allows to quickly copy the response into clipboard)
2. Buttons were made larger
3. Query response time is color-coded now depending on how slow or fast any given query is.
4. We've disabled syntax highlighting in the query response as a workaround for syntax highlighting performance being severely affected by wp-emoji's MutationObserver.

Highlighting large JSON blobs results in a lot of HTML nodes which, in conjunction, with wp-emoji MutationObserver makes it's extremely slow, to the point of being unusable. 

The proper fix is to provide API in core wp-emoji to allow disabling the MutationObserver

### Plugin Updated: Search Dev Tools

1. A copy button is implemented (allows to quickly copy the response into clipboard)
2. Buttons were made larger
3. Query response time is color-coded depending on how slow or fast any given query is.
4. We've disabled syntax highlighting in the query response as a workaround for syntax highlighting performance being severely affected by wp-emoji's MutationObserver.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [ ] I've created a changelog description that aligns with the provided examples.

### CODE REVIEW

File: search/search-dev-tools/src/style/prism.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		white-space: pre;
 		word-spacing: normal;
 		word-break: normal;
-		font-size: 0.9em;
+		font-size: 12px;
```
Comment (sjinks) [inline search/search-dev-tools/src/style/prism.scss]: Maybe we should try to find the proper `em` value? `em`s scale better if the user wants to zoom in or zoom out

File: search/search-dev-tools/src/components/query/index.js [hunk 2]
```diff
@@ -72,9 +72,9 @@ const Query = ( { args, request, url, query_args, backtrace = [] } ) => {
 	}, [ state.query, state.editing ] );
 
 	// Re-highlight the query on result change (after the response is received).
-	useEffect( () => {
-		highlightElement( queryResultRef.current );
-	}, [ queryResultRef, state.result ] );
+	// useEffect( () => {
+	// 	highlightElement( queryResultRef.current );
+	// }, [ queryResultRef, state.result ] );
```
Comment (sjinks) [inline search/search-dev-tools/src/components/query/index.js]: I would suggest that we remove the commented-out code - we can always restore it from Git.

Comment (sjinks) [reply search/search-dev-tools/src/components/query/index.js]: Also, `highlightElement` on line 5 also needs to be removed.

File: search/search-dev-tools/src/style/prism.scss [hunk 3]
```diff
@@ -28,7 +28,7 @@
 	}
 
 	pre > code[class*='language-'] {
-		font-size: 1em;
+		// font-size: 1em;
```
Comment (sjinks) [inline search/search-dev-tools/src/style/prism.scss]: I would suggest to remove the entire CSS rule :-)

**Review by sjinks — CHANGES_REQUESTED** [review]

File: search/search-dev-tools/src/style/prism.scss (hunk 3 above)
Comment (rinatkhaziev) [reply search/search-dev-tools/src/style/prism.scss]: thanks, sloppy Friday afternoon coding right here :)

**Review by sjinks — DISMISSED** [review]

**Review by sjinks — APPROVED** [review]

**Pull request merged**